### ISSUE TITLE
Convention Proposal: Unnecessary UTB Trash Push

### ISSUE THREAD

**Issue by Dr-Kakashi** [Zamiell/hanabi-conventions#235]
![2020-05-25_2115](https://user-images.githubusercontent.com/48993588/82859948-0f46bd80-9ecd-11ea-89e7-264716ad56a4.png)

Trash bluff on Lilliana
Lilliana plays B2 turning the b1 on Kimbi into trash
Kyle knows he has R1.  Kyle also realizes that if he does nothing, Kimbi will trash push.  Kyle plays R1
Kimbi can mark b1 as trash, but she wonders to herself what's the purpose of giving her trash on chop.
She can also understand that it was unnecessary as b2 can easily be clued.
She interprets this as a trash push on her and plays R2

Basically it's the Unnecessary trash push with a unknown trash component
